content/200-concepts/100-components/03-prisma-migrate/010-mental-model.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: 'A mental model for Prisma Migrate'
+metaTitle: 'A mental model for Prisma Migrate'
+metaDescription: 'A mental model guide for working with Prisma Migrate in your project'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This guide provides a conceptual overview of database migrations using Prisma Migrate when working with relational databases. It covers: what database migrations are, their value, and what Prisma Migrate is and how you can evolve your database schema with Prisma Migrate in different environments.
+
+**This guide does not apply to MongoDB** because Prisma Migrate only supports relational databases.
+
+</TopBlock>
+
+## What are database migrations?
+
+Database migrations are a controlled set of changes that modify and evolve the structure of your database schema. Migrations help you transition your database schema from one state to another. For example, within a migration you can create or remove tables and columns, split fields in a table, or add types and constraints to your database.
+
+### Patterns for evolving database schemas
+
+This section describes general schema migration patterns that exist for evolving database schemas.
+
+The two main schema migration patterns are:
+
+- **Model/Entity-first migration:** with this pattern, you define the structure of the database schema with code and then use a migration tool to generate the SQL, for example, for syncing your application and database schema.
+
+![](https://user-images.githubusercontent.com/33921841/194377029-305dc60b-f651-46e4-87f1-508884ebd972.png)
```

Comment:
General comment for all images: I do not think we host our images on GitHub. Someone from docs should be able to explain what we do instead.

Reply:
btw, are we planning to have prettified versions of these @ruheni?

Reply:
I think we should. I'll create a design task for them.

Reply:
Current docs guidelines for images are here, looks like we're saving them in the same directory: https://www.prisma.io/docs/about/prisma-docs/style-guide/image-guidelines